e2e-tests/cypress/integration/appeal-submission-appeal-site-address.feature

```diff
@@ -53,7 +51,7 @@ Feature: Appellant provides the Appeal Site Address
 
     Scenario Outline: Prospective appellant provides address data that exceeds the maximum length constraint for each field
         Given the user is prompted for the site address
-        When the user provides a value which is too long (<component>: <count>)
+        When the user provides a value which is too long - <component> : <count>
         Then the user is informed that they cannot continue with the provided address because <reason>
         #And the user can see that their appeal has NOT been updated with the provided address
```

Comment:
This looks like an important behaviour. Should it be commented out?

Reply:
I assumed that it was while we were waiting for the check your answer page. So I just took the feature file as it is

Reply:
I think we should be able to confirm the updated state without the check your answers page.